vitest.config.ts

```diff
@@ -0,0 +1,37 @@
+import path from "node:path";
```

Comment:
Leaving a note here that the Rollup configuration is trying to bundle this (or it's the type definitions generation step). Can you take a peek to see if there's a reason why? It's causing `lib` to include `vitest.config.d.ts`, which is not something we should be shipping since it's internal.